### ISSUE TITLE
Quests display bug

### ISSUE THREAD

**Issue by Fatinasp** [KC3Kai/KC3Kai#1521]
Hello

My quests displays is completely broken since I finish the 1st part of the "new" furniture quest (for the land base aircraft). The bug happen between the implementation of the quest and the translation in KC3 (when I finish it was display in japanese). I don't mind it because there was only 1 quest slot that  bugged but now all my furniture quests are bug and I can't remove them.

![sans titre-1](https://cloud.githubusercontent.com/assets/21972323/18225573/ac016990-7230-11e6-9b98-1353244e2843.png)
As you see, I have only 2 quests active but there is many quests that can be seen even if I finish them.

Chrome version: 52.0.2743.116 (64 bit)
KC3kai version: 30.9.0

**Comment by dragonjet** [comment]
click on the boxes on the already completed quests (those with green borders) then go back to home port screen

**Comment by nonsensed** [comment]
This seems to be a prevalent issue with a number of users. While I'm worried I somehow caused this when I TL'd the EN quests.json, I don't see anything specific that I did wrong. I could be blind, though. No idea if this happens for other TL languages.

I don't have the quests anymore, but it seems when turning in the new the land bomber quests, kc3 doesn't automatically remove them after turning them in. When I did them with the EN TL done, in the quests->flowchart screen, I could see an untranslated quest and the translated quest which I had to remove manually. I didn't think at the time to check the console for any errors.

So this could be either a golden opportunity to kick my ass for doing something stupid, or there's some general problem with the quests processing.
